service-runner/src/main/resources/config/stellar.localhost.toml

```diff
@@ -18,6 +18,14 @@ is_asset_anchored = false
 desc = "Circle USDC Token"
 anchor_asset_type = "crypto"
 
+[[CURRENCIES]]
```

Comment:
We should avoid duplicated assets.

Reply:
They added the real testnet USDC, which I think is better to use than a custom USDC token we issued. Can we remove our own custom asset instead of removing Circle's?

Reply:
Should we update all end2End/integration/unit tests to use `stellar:USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5` instead of `stellar:USDC:GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP?

But there may be a problem if accounts, that are used in tests, do not  have a trustline configured for this asset. It can affect end2End tests. But we can configure trustline, if there are secret keys for all these accounts stored in test config files.